examples/email-recovery/src/pages/index.tsx

```diff
@@ -0,0 +1,239 @@
+  const [iframeStamper, setIframeStamper] = useState<IframeStamper | null>(
+    null
+  );
+  const {
+    register: initRecoveryFormRegister,
+    handleSubmit: initRecoveryFormSubmit,
+  } = useForm<InitRecoveryFormData>();
+  const {
+    register: recoverUserFormRegister,
+    handleSubmit: recoverUserFormSubmit,
+  } = useForm<RecoverUserFormData>();
+
+  const initRecovery = async (data: InitRecoveryFormData) => {
+    const response = await axios.post("/api/initRecovery", {
+      email: data.email,
+      targetPublicKey: data.targetPublicKey,
+    });
+    setInitRecoveryResponse(response.data);
+  };
+
+  const recoverUser = async (data: RecoverUserFormData) => {
+    if (iframeStamper === null) {
+      throw new Error("iframeStamper is null");
+    }
+    if (initRecoveryResponse === null) {
+      throw new Error("initRecoveryResponse is null");
+    }
+
+    let injected = await iframeStamper.injectRecoveryBundle(
+      data.recoveryBundle
+    );
+    if (injected !== true) {
+      throw new Error("unexpected error while injecting recovery bundle");
+    }
+
+    const challenge = generateRandomBuffer();
+    const authenticatorUserId = generateRandomBuffer();
+
+    // An example of possible options can be found here:
+    // https://www.w3.org/TR/webauthn-2/#sctn-sample-registration
+    const attestation = await getWebAuthnAttestation({
+      publicKey: {
+        authenticatorSelection: {
+          residentKey: "preferred",
+          requireResidentKey: false,
+          userVerification: "preferred",
+        },
+        rp: {
+          id: "localhost",
+          name: "Turnkey Federated Passkey Demo",
+        },
+        challenge,
+        pubKeyCredParams: [
+          { type: publicKey, alg: es256 },
+          { type: publicKey, alg: rs256 },
+        ],
+        user: {
+          id: authenticatorUserId,
+          name: data.authenticatorName,
+          displayName: data.authenticatorName,
+        },
+      },
+    });
+
+    const client = new TurnkeyClient(
+      {
+        baseUrl: process.env.NEXT_PUBLIC_BASE_URL!,
+      },
+      iframeStamper
+    );
+
+    // TODO: switch this to be `client.recoverUser`!
+    // Similar in spirit though, we're adding a new authenticator.
+    const signedRequest = await client.stampCreateAuthenticators({
+      type: "ACTIVITY_TYPE_CREATE_AUTHENTICATORS_V2",
+      timestampMs: String(Date.now()),
+      organizationId: initRecoveryResponse.organizationId,
+      parameters: {
+        userId: initRecoveryResponse.userId,
+        authenticators: [
+          {
+            authenticatorName: data.authenticatorName,
+            challenge: base64UrlEncode(challenge),
+            attestation: attestation,
+          },
+        ],
+      },
+    });
+
+    // TODO: instead of simply alerting, POST the request to Turnkey to complete recovery.
+    console.log("SUCCESS", signedRequest);
+    alert("SUCCESS! Request SIGNED!");
```

Comment:
should we direct attention in the alert to the console?

Reply:
I've reworked this part to actually POST. Resolved :)